### PR TITLE
[New3D] Make camera_info topic matching more lenient

### PR DESCRIPTION
**Pull request by jtbandes**
**User-Facing Changes**
Improved the automatic matching between camera images and camera info topics in the 3D (Beta) panel.

**Description**
Fixes https://github.com/foxglove/studio/issues/3898

### CODE REVIEW

File: packages/studio-base/src/panels/ThreeDeeRender/color.ts [hunk 1]
```diff
@@ -65,9 +64,9 @@ export function rgbaToCssString(color: ColorRGBA): string {
 }
 
 export function rgbaToLinear(output: ColorRGBA, color: ColorRGBA): ColorRGBA {
-  output.r = SRGBToLinear(color.r);
-  output.g = SRGBToLinear(color.g);
-  output.b = SRGBToLinear(color.b);
+  output.r = THREE.SRGBToLinear(color.r);
+  output.g = THREE.SRGBToLinear(color.g);
+  output.b = THREE.SRGBToLinear(color.b);
```
Comment (amacneil) [inline packages/studio-base/src/panels/ThreeDeeRender/color.ts:69]: unrelated or accidental?

File: packages/studio-base/src/panels/ThreeDeeRender/color.ts [hunk 2]
```diff
@@ -2,14 +2,13 @@
 // License, v2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/
 
-import { SRGBToLinear } from "three/src/math/ColorManagement";
-import { clamp } from "three/src/math/MathUtils";
+import * as THREE from "three";
```
Comment (amacneil) [inline packages/studio-base/src/panels/ThreeDeeRender/color.ts:5]: Why not `import { SRGBToLinear, MathUtils } from "three"` ?

**Review by amacneil — APPROVED** [review]
cool, works with my test bag

File: packages/studio-base/src/panels/ThreeDeeRender/color.ts (hunk 1 above)
Comment (jtbandes) [reply packages/studio-base/src/panels/ThreeDeeRender/color.ts:69]: Had to change how the imports are done to make it work in jest

File: packages/studio-base/src/panels/ThreeDeeRender/color.ts (hunk 2 above)
Comment (jtbandes) [reply packages/studio-base/src/panels/ThreeDeeRender/color.ts:5]: 🤷  we could, but I think THREE is a pretty standard way of using three.js

Comment (defunctzombie) [reply packages/studio-base/src/panels/ThreeDeeRender/color.ts:5]: Kinda weird that this import is special and given all caps we use for file level constants.

**Pull request merged**